src/app/shared/ui/header/header.component.html

```diff
@@ -0,0 +1,23 @@
+<header>
+  <div class="header__close-modal-button">
+    <mat-toolbar color="primary">
+      <button
+        mat-icon-button
+        class="example-icon"
+        class="header__notifications-btn"
+        aria-label="Example icon-button with menu icon">
+        <mat-icon>menu</mat-icon>
+      </button>
+      <span class="header__title">{{ title }}</span>
+      <span class="header__example-spacer"></span>
+      <button mat-icon-button class="header__notifications-btn">
+        <mat-icon>notifications</mat-icon>
+      </button>
+      <app-user-initials-button class="header__initials" [cdkMenuTriggerFor]="menu"></app-user-initials-button>
```

Comment:
z tego co wiem, to dobrą praktyką jest ustawiać w selektorach  rzeczy w kolejności od ważniejszych, do najmniej ważnych, czyli w tym przypadku najpierw [cdkMenuTriggerFor] a dopiero pozniej class;  w innych msc podobna uwaga